app/policies/teacher_policy.rb

```diff
@@ -1,4 +1,26 @@
 class TeacherPolicy < ApplicationPolicy
+
+  def index?
+    user.admin?
+  end
+
+  def destroy?
+    index?
+  end
+
+  def new?
+    index?
+  end
+
+  def create?
+    index?
+  end
+
+  def update?
+    index?
+  end
+
+  # TODO: this seems wrong
```

Comment:
@erinnachen see here. I dont think we want to use the teacher policy for the grove monitor. we probably need our own policy that we use for those endpoints. just because a teacher is logged in and is a teacher, doesn't mean they should be able to see a teacher in another school or grove, which is what this policy action states.